sql-database/failover-groups/add-single-db-to-failover-group-az-cli.sh

```diff
@@ -0,0 +1,112 @@
+﻿#!/bin/bash
```

Comment:
This script contains a Byte-Order Marker (BOM). This is indicated by the following output:

```bash
$> head -1 add-single-db-to-failover-group-az-cli.sh | od -c
0000000 357 273 277   #   !   /   b   i   n   /   b   a   s   h  \n
0000017
```

See https://unix.stackexchange.com/questions/27054/bin-bash-no-such-file-or-directory/27067

You need to remove the BOM on this file. That can be done by running the `dos2unix` command on Linux systems, or by using Notepad++ on Windows (see https://stackoverflow.com/questions/32986445/remove-a-bom-character-in-a-file)

Reply:
> The BOM issue is still present and **must** be fixed before merge. See [#261 (comment)](https://github.com/Azure-Samples/azure-cli-samples/pull/261#discussion_r302689178)
> 
> All other reported issues should also be fixed before merge.

i tried doing what it says in your post by using notepad++ so i'm not sure what else i can do to resolve this. can you please help with this?